src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Unix.cs

```diff
@@ -168,6 +173,7 @@ protected internal override Task<HttpResponseMessage> SendAsync(HttpRequestMessa
         #region Private
 
         private readonly CurlHandler _curlHandler;
+        private readonly DiagnosticsHandler _diagnosticsPipeline;
```

Comment:
Why instanced diagnostics handler object? It doesn't appear to contain any state.